router/lambda.js

```diff
@@ -34,7 +34,11 @@ const authoriseHandler = (req, res, next) => {
   const isAuthenticated = checkAuth.execute({
     token: req.cookies.hackneyToken
   });
-  if (!isAuthenticated && req.url !== '/loggedout') {
+  if (
+    !isAuthenticated ||
+    (process.env.TOKEN_RESET_IAT > checkAuth.getIat(req.cookies.hackneyToken) &&
+      req.url !== '/loggedout')
+  ) {
     res.writeHead(302, { Location: '/loggedout' });
```

Comment:
So when we redirect a user with an existing token to the `/loggedout` page, does this reset their token?

Reply:
Jira ticket discussion was mentioning the redirection to: **https://auth.hackney.gov.uk/auth?redirect_uri=https://frontdoor-snapshot.hackney.gov.uk** in order to achieve this.

Reply:
Yes you're right! This would take them through the login process... redirecting to this url for me took me directly to the google login screen instead of doing it silently, so I thought it might feel a bit confusing for someone going onto BC to end up on an authentication page straightaway.
There might be a different way to do this so that it's silent, but I'm wondering how useful it would be/how often would those groups change

Reply:
This is also intended as a one off solution and if a new feature came out that needed those tokens immediately reset we'd have to change the token reset env var. 
Another way we could do it is by rotating token every time they visit (or daily) and doing it silently if possible

Reply:
I see, it makes sense. Yeah, I agree the silent reset is a nice to have. If it can be done quick, it would be nice to have. Otherwise, I think this PR is good to go.